### DISCUSSION TITLE
Slow connection and even timeouts on github.com

### DISCUSSION THREAD

**Discussion by kruzo7** [community/community#128161] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello

I'm facing slow connection and even timeouts to github.com

Tested with ip4 and ip6 - same problem

Results from https://github-debug.com/

[github-log.txt](https://github.com/user-attachments/files/15793163/github-log.txt)

**Comment by vishal11u** [comment]
Hi, Have you tried using different networks or devices to see if the issue persists? Additionally, you could try clearing your browser cache or using a different web browser to see if that helps. If the problem persists, you might want to reach out to your internet service provider to see if there are any network issues on their end.

**Comment by kruzo7** [comment]
Checked on Firefox and Chromium.

On Firefox I had excluded domain github.com in privacy settings but that didn't help.

Well, other web pages are working fine and seems that when I'm logged in github problem don't exists.
